demo/grid/index.html

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html>
+<head>
+  <title>Grid / CSS Components / Zendesk Garden</title>
+  <link href="../bedrock/index.css" rel="stylesheet">
+  <link href="../menus/index.css" rel="stylesheet">
+  <link href="../forms/checkbox/index.css" rel="stylesheet">
+  <link href="//zendeskgarden.github.io/index.css" rel="stylesheet">
+  <link href="index.css" rel="stylesheet">
+  <link href="../utilities/index.css" rel="stylesheet">
+</head>
+<body>
+  <header class="c-header">
+    <figure class="c-header__logo">
+      <a class="u-fg-inherit" href="//zendeskgarden.github.io">
+        <svg class="c-header__logo__svg svg-square">
+          <use xlink:href="../index.svg#zd-svg-icon-26-zendesk" />
+        </svg>
+      </a
+    ></figure
+    ><h1 class="c-header__title"
+      ><a class="u-fg-inherit" href="..">CSS Components</a>
+      <span>/ Grid</span>
+    </h1>
+    <ul class="c-ctl c-ctl--header">
+      <li class="c-ctl__item u-position-relative">
+        <a class="c-menu-ctl js-menu" href="#settings">
+          <svg>
+            <use xlink:href="../index.svg#zd-svg-icon-26-settings-fill" />
+          </svg>
+          <span>Settings</span>
+          <svg>
+            <use xlink:href="../index.svg#zd-svg-icon-14-chevron" />
+          </svg>
+        </a>
+        <ul aria-hidden="true" class="c-menu c-menu--down" role="menu">
+          <li class="c-menu__item" role="menuitem">
+            <fieldset class="c-chk">
+              <input class="c-chk__input js-rtl" id="nav.ctl.rtl" type="checkbox">
+              <label class="c-chk__label c-chk__label--toggle" for="nav.ctl.rtl">RTL</label>
+            </fieldset>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </header>
+  <main class="c-main c-main--page">
+    <h1 class="c-main__title">Grid</h1>
+    <div class="c-main__body">
+      <p class="u-mb"><code class="c-code">@zendeskgarden/css-grid</code> is an opinionated customization of the Bootstrap v4 flexbox grid.
+        All classes are consistent with the
+        <a href="https://getbootstrap.com/docs/4.0/layout/grid/">Bootstrap Grid Documentation.</a>
+      </p>
+      <h2 class="u-gamma u-mb">Mobile-Responsive Columns</h2>
+      <div class="container-fluid debug u-mb-xl">
+        <div class="row">
+          <div class="col-md">
+            one of three columns
+          </div>
+          <div class="col-md">
+            two of three columns
+          </div>
+          <div class="col-md">
+            three of three columns
+          </div>
+        </div>
+      </div>
+      <h2 class="u-gamma u-mb">RTL Locales</h2>
+      <p class="u-mb">
+        You can enable RTL layouts by using the <code class="c-code">.is-rtl</code> class with the
+        <code class="c-code">.container</code> or <code class="c-code">.container-fluid</code>
+        wrapper classes</a>.
+      </p>
+      <div class="container-fluid is-rtl debug u-mb-xl">
```

Comment:
This section doesn't make sense after a user has toggled the page's `RTL` on/off. Maybe just move the paragraph text up to the top and call out the ability to toggle RTL on the page.

Reply:
Moved.